src/Tools/Microsoft.VisualStudio.Templates.Editorconfig.Wizard/Generator/RoslynEditorConfigFileGenerator.cs

```diff
@@ -1,56 +1,22 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
 
 using EnvDTE80;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Options;
-using Microsoft.VisualStudio.ComponentModelHost;
-using Microsoft.VisualStudio.LanguageServices;
-using Microsoft.VisualStudio.RpcContracts.FileSystem;
 using Microsoft.VisualStudio.Shell;
-using Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Logging.Kinds;
-using Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Logging.Messages;
 using System;
-using System.Collections.Immutable;
-using System.IO;
 using System.Reflection;
 using static Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Logging.Logger;
+using Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator.Api;
 
 namespace Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Generator;
 
 public class RoslynEditorConfigFileGenerator
 {
-    private const BindingFlags PublicBindingFlags = BindingFlags.Instance | BindingFlags.Static | BindingFlags.Public;
-    private const BindingFlags NonPublicBindingFlags = BindingFlags.Instance | BindingFlags.Static | BindingFlags.NonPublic;
-
     public RoslynEditorConfigFileGenerator(DTE2 dte)
     {
         try
         {
-            string? basePath = Path.GetDirectoryName(dte.FullName);
-            string languageServiceSubPath = @"CommonExtensions\Microsoft\VBCSharp\LanguageServices";
-            var baseDirectory = Path.Combine(basePath!, languageServiceSubPath);
-
-            _microsoftVisualStudioLanguageServicesVisualBasicAssembly =
-                new Lazy<Assembly>(() => Assembly.LoadFrom(Path.Combine(baseDirectory, "Microsoft.VisualStudio.LanguageServices.VisualBasic.dll")));
-
-            _microsoftVisualStudioLanguageServicesCSharpAssembly =
-                new Lazy<Assembly>(() => Assembly.LoadFrom(Path.Combine(baseDirectory, "Microsoft.VisualStudio.LanguageServices.CSharp.dll")));
-
             _serviceProvider = new ServiceProvider(dte as OLE.Interop.IServiceProvider);
-
-            _generatorMethod = new Lazy<MethodInfo?>(() =>
-            {
-                var editorConfigFileGeneratorType = typeof(OptionSet).Assembly.GetType("Microsoft.CodeAnalysis.Options.EditorConfigFileGenerator");
-                Type[] targetParams = new Type[]
-                {
-                typeof(ImmutableArray<(string feature, ImmutableArray<IOption> options)>),
-                typeof(OptionSet),
-                typeof(string)
-                };
-
-                return editorConfigFileGeneratorType?.GetMethod("Generate", PublicBindingFlags, binder: null, types: targetParams, modifiers: null);
-            });
         }
         catch (Exception ex)
         {
```

Comment:
"Roslyn" has a typo on the line below this. Not your change but if you're here...(although we can really just delete this...)